modules/virt-disable-auto-updates-single-boot-source.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-disable-auto-updates-single-boot-source_{context}"]
+= Disabling automatic updates for a single boot source
+
+You can disable automatic updates for an individual boot source, whether it is custom or system-defined, by editing the `HyperConverged` custom resource (CR).
+
+.Procedure
+
+. Open the `HyperConverged` CR in your default editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Disable automatic updates for an individual boot source by editing the `spec.dataImportCronTemplates` field.
+
+* To disable automatic updates for a custom boot source, remove the boot source from the `spec.dataImportCronTemplates` field. Automatic updates are disabled for custom boot sources by default.
+
+* To disable automatic updates for a system-defined boot source:
```

Comment:
Is there a way to avoid having the sentence completion dangle into the sub-steps?

Reply:
Great point; I'm trying this (feedback welcome):
```
. Disable automatic updates for an individual boot source by editing the `spec.dataImportCronTemplates` field.
+
Custom boot source::
* Single step

+
System-defined boot source::
.. Step 1
.. Step 2
```

Reply:
Seems good to me!